src/assets/thirdPartyServices/snet/yolov3_object_detection/index.js

```diff
@@ -4,40 +4,50 @@ import Button from "@material-ui/core/Button";
 import Slider from "@material-ui/lab/Slider";
 
 import SNETImageUpload from "../../standardComponents/SNETImageUpload";
+import { Detector } from "./object_detection_pb_service";
```

Comment:
The service is called `Detect` I guess. Can we quickly verify that.

Reply:
This should not impact as we are not using for the methods. It should be same when we are calling the methodNames otherwise we can any name.

Anyway will change it to be consistent.